.travis.yml

```diff
@@ -4,50 +4,65 @@ cache: ccache
 
 matrix:
   include:
-  #Ubuntu
-  - name: Ubuntu (Debug)
+  #Ubuntu Bionic (on docker)
+  - name: Ubuntu Bionic (Debug)
     if: tag IS NOT present
-    os: linux
-    dist: xenial
-    group: stable
-    env: BUILDTYPE=Debug
-  - name: Ubuntu (Release)
+    services: docker
+    before_install: docker build -t img .ci
+    script: docker run --mount "type=bind,source=$(pwd),target=/src" -w="/src" img
+          bash .ci/travis-compile.sh --server "Debug"
+  - name: Ubuntu Bionic (Debug)
     if: (branch = master AND NOT type = pull_request) OR tag IS present
+    services: docker
+    script: docker build -t img .ci &&
+        docker run --mount "type=bind,source=$(pwd),target=/src" -w="/src" img
+          bash .ci/travis-compile.sh --server "Release"
```

Comment:
Do we need `--package` here, so that it can be uploaded it to the github release?